international-journal-of-geriatric-psychiatry.csl

```diff
@@ -160,7 +159,7 @@
       </group>
     </layout>
   </citation>
-  <bibliography>
+  <bibliography delimiter-precedes-et-al="always" et-al-min="4" et-al-use-first="3">
```

Comment:
I think this is 6/3 or 7/3 I see at least 5 listed authors here: http://onlinelibrary.wiley.com/doi/10.1002/gps.4447/epdf I think 7/3 is the most common rule, so I'd go with that.